pages/plugins.jsx

```diff
@@ -33,23 +21,31 @@ const integrations = [
   },
 ]
 
-const Plugins = () => {
-  return (
-    <>
-      <h1>Plugins</h1>
-      <p>Integrate our powerful services into your website.</p>
-      <div className={pluginsClassNames.plugins}>
-        {integrations.map(i => (
-          <Plugin
-            key={i.name}
-            name={i.name}
-            abstract={i.abstract}
-            description={i.description}
-          />
-        ))}
-      </div>
-    </>
-  )
+class Plugins extends React.Component {
+  static async getInitialProps({ initialStoreData }) {
+    const { statusCode, ...plugins } = await API.getIntegrations(140)
+    initialStoreData.plugins = plugins
+  }
```

Comment:
Yeah, we can start from something like this. However, pages are presentation so we will need to move it somewhere unless I change my mind 😅

Reply:
Yeah, but everyone knows it's under development. I don't see any reason for hiding it. 😄

Reply:
Architecture should be considered from the beginning. In this case this is not a big deal but we have to sort it out somehow later.